src/cts/.gitignore

```diff
@@ -1,2 +1,3 @@
 test/src
 test/results
+test/cts.clk.buffer
```

Comment:
What is this file?

Reply:
I think this comes from 
```
set_debug_level CTS legalizer 3
```
in check_buffers_blockage.tcl.  I think you could lower that to 2 and avoid this file being written.  It isn't used in the test.